index.js

```diff
@@ -1,13 +1,14 @@
 'use strict';
 var through = require('through2');
 
-module.exports = function () {
+module.exports = function (reversions) {
+	reversions = reversions || 1;
 	return through.obj(function (file, enc, cb) {
 		var history = file.history;
 
-		if (history.length > 1) {
+		if (history.length > reversions) {
```

Comment:
reverse as much as possible? Since it's backward compatible any no one raised a request on throwing an error before.